tree-construction/webkit02.dat

```diff
@@ -438,34 +439,34 @@ eof-in-math
 |     <select>
 |       <optgroup>
 |         <option>
-|       <hr>
+|         <hr>
```

Comment:
This should be reverted per https://github.com/whatwg/html/pull/10557#discussion_r1828988927

Reply:
done